cmd/sblookup/main.go

```diff
@@ -77,9 +78,10 @@ func main() {
 		os.Exit(codeInvalid)
 	}
 	sb, err := safebrowsing.NewSafeBrowser(safebrowsing.Config{
-		APIKey: *apiKeyFlag,
-		DBPath: *databaseFlag,
-		Logger: os.Stderr,
+		APIKey:    *apiKeyFlag,
+		DBPath:    *databaseFlag,
+		Logger:    os.Stderr,
+		ServerURL: *serverUrlFlag,
```

Comment:
Just caught that. Too eager! Fixed in https://github.com/google/safebrowsing/pull/46/commits/fa634358c5b95f982747f4d1c23781432ef19c59